loadtest/utils_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package loadtest
+
+import (
+	"testing"
+	"time"
+)
+
+func TestRandomFutureTime(t *testing.T) {
+	deltaStart := 10 * time.Second
+	maxUntil := 5 * time.Minute
+
+	now := time.Now()
+	start := now.Add(deltaStart)
+	end := start.Add(maxUntil)
+
+	randomTime := RandomFutureTime(deltaStart, maxUntil)
+
+	if randomTime < start.Unix() || randomTime > end.Unix() {
+		t.Errorf("RandomFutureTime() = %v, want between %v and %v", randomTime, start.Unix(), end.Unix())
+	}
```

Comment:
We may want to use the `require` package for these checks.

Reply:
Done.